bower.json

```diff
@@ -1,15 +1,15 @@
 {
   "name": "enyo-strawman",
   "dependencies": {
-    "enyo": "git@github.com:enyojs/enyo.git#2.6.0-pre.17.1",
-    "moonstone": "git@github.com:enyojs/moonstone.git#2.6.0-pre.17.1",
-    "spotlight": "git@github.com:enyojs/spotlight.git#2.6.0-pre.17.1",
-    "enyo-ilib": "git@github.com:enyojs/enyo-ilib.git#2.6.0-pre.17.1",
-    "layout": "git@github.com:enyojs/layout.git#2.6.0-pre.17.1",
-    "enyo-cordova": "git@github.com:enyojs/enyo-cordova.git#2.6.0-pre.17.1",
-    "enyo-webos": "git@github.com:enyojs/enyo-webos.git#2.6.0-pre.17.1",
-    "onyx": "git@github.com:enyojs/onyx.git#2.6.0-pre.17.1",
-    "canvas": "git@github.com:enyojs/canvas.git#2.6.0-pre.17.1",
+    "enyo": "git@github.com:enyojs/enyo.git#2.6.0-pre.18.1",
+    "moonstone": "git@github.com:enyojs/moonstone.git#2.6.0-pre.18.1",
+    "spotlight": "git@github.com:enyojs/spotlight.git#2.6.0-pre.18.1",
+    "enyo-ilib": "git@github.com:enyojs/enyo-ilib.git#2.6.0-pre.18.1",
+    "layout": "git@github.com:enyojs/layout.git#2.6.0-pre.18.1",
+    "enyo-cordova": "git@github.com:enyojs/enyo-cordova.git#2.6.0-pre.18.1",
+    "enyo-webos": "git@github.com:enyojs/enyo-webos.git#2.6.0-pre.18.1",
+    "onyx": "git@github.com:enyojs/onyx.git#2.6.0-pre.18.1",
+    "canvas": "git@github.com:enyojs/canvas.git#2.6.0-pre.18.1",
```

Comment:
Turns out we no longer need `bower.json` and more. We can safely delete this file altogether.